src/jarque_bera.jl

```diff
@@ -0,0 +1,69 @@
+# jarque_bera.jl
+# Jarque-Bera goodness-of-fit test
+#
+# Copyright (C) 2017   Benjamin Born
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+# LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+export JarqueBeraTest
+
+immutable JarqueBeraTest <: HypothesisTest
+    n       ::Int           # number of observations
+    JB      ::Float64       # test statistic
+    m3      ::Float64       # skewness
+    m4      ::Float64       # excess kurtosis
```

Comment:
I took the vertical alignment advice from here: [https://github.com/JuliaPraxis/Spacing/blob/master/guides/VerticalSpacing.md](https://github.com/JuliaPraxis/Spacing/blob/master/guides/VerticalSpacing.md) but agree that it was excessive.